modules/light-clients/08-wasm/testing/simapp/upgrades.go

```diff
@@ -0,0 +1,44 @@
+package simapp
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	circuittypes "cosmossdk.io/x/circuit/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+const (
+	IBCWasmUpgrade = "ibcwasm-v8"
+)
+
+// registerUpgradeHandlers registers all supported upgrade handlers
+func (app *SimApp) registerUpgradeHandlers() {
+	app.UpgradeKeeper.SetUpgradeHandler(
+		"ibcwasm-v8",
+		createWasmStoreUpgradeHandler(app.ModuleManager, app.configurator),
+	)
+
+	upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
+	if err != nil {
+		panic(err)
+	}
+
+	if upgradeInfo.Name == "ibcwasm-v8" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{
+				circuittypes.ModuleName,
```

Comment:
I kept this the same as the ibc-go/v8 simapp upgrade. I will check the v7.x release branch and ensure that x/circuit is not included in v7 branch but _is_ in v8 - I reckon this is probably still needed